cli/cobra.go

```diff
@@ -159,15 +161,31 @@ func isFirstLevelCommand(cmd *cobra.Command) bool {
 	return cmd.Parent() == cmd.Root()
 }
 
-func commandVendor(cmd *cobra.Command) string {
+func isPluginFirstLevelCommand(cmd *cobra.Command) bool {
+	return isPlugin(cmd) && isFirstLevelCommand(cmd)
+}
+
+func decoratedCommandName(cmd *cobra.Command) string {
+	decoration := " "
+	if isPluginFirstLevelCommand(cmd) {
+		decoration = "*"
+	}
+	return cmd.Name() + decoration + strings.Repeat(" ", cmd.NamePadding()-len(cmd.Name()))
```

Comment:
Can you use `.rpad decoratedCOmmandName .NamePadding` in the template in preference to padding here? If so that would be better I think.

Reply:
Done... Note that I had to add a function to have the separation (command->description) space